Sparkle/SUUpdatePermissionPrompt.m

```diff
@@ -52,7 +52,7 @@ @implementation SUUpdatePermissionPrompt
 
 - (BOOL)shouldAskAboutProfile
 {
-    return [[self.host objectForInfoDictionaryKey:SUEnableSystemProfilingKey] boolValue];
+    return [(NSString *)[self.host objectForInfoDictionaryKey:SUEnableSystemProfilingKey] boolValue];
```

Comment:
Hm. I failed to trace the code far enough here to find out if the object should be expected to be an NSString or an NSNumber. Did you find out?

Reply:
I couldn't find where it was set so I figured casting to a string wouldn't do any harm, even if the value of the key is a number.